gojira.sh

```diff
@@ -819,7 +819,12 @@ main() {
     p_compose down -v
     ;;
   shell)
-    p_compose exec --index $GOJIRA_CLUSTER_INDEX $GOJIRA_TARGET gosh -l -i
+    local opt_envvars
+    # if requested, we set environment variables
+    for binding in "${_GOJIRA_CLI_ENVS[@]}"; do
+      opt_envvars+="  --env ${binding}"
+    done
```

Comment:
```
${_GOJIRA_CLI_ENVS[*]/#/'--env '}
```

Reply:
utilized in https://github.com/Kong/gojira/pull/10/commits/7072eb92c330db09aacf286413b00311f16d3fb9